chain-impl-mockchain/src/fragment/mod.rs

```diff
@@ -36,6 +36,7 @@ pub enum Fragment {
     VoteCast(Transaction<certificate::VoteCast>),
     VoteTally(Transaction<certificate::VoteTally>),
     EncryptedVoteTally(Transaction<certificate::EncryptedVoteTally>),
+    SmartContractDeploy(Transaction<smartcontract::Contract>),
```

Comment:
Should be renamed to just `Evm` or something like that maybe.

Reply:
This is done.